lib/cadet/assessments/assessments.ex

```diff
@@ -524,10 +525,12 @@ defmodule Cadet.Assessments do
   def update_grading_info(
         %{submission_id: submission_id, question_id: question_id},
         attrs,
-        grader = %User{role: role}
+        grader = %User{id: grader_id, role: role}
       )
       when is_ecto_id(submission_id) and is_ecto_id(question_id) and
              (role in @grading_roles or role in @see_all_submissions_roles) do
+    attrs = Map.put(attrs, "grader_id", grader_id)
```

Comment:
Why not use `:grader_id` as key?

Reply:
all keys in attr are strings not atoms? It throws this error when i try to use an atom instead.

```
 ** (Ecto.CastError) expected params to be a map with atoms or string keys, got a map with mixed keys: %{:grader_id => 48101, "adjustment" => -100, "comment" => "Your awesome", "xpAdjustment" => -100, "xp_adjustment" => -100}

```

Reply:
Right, my bad.